test/server/impl/base_test.py

```diff
@@ -0,0 +1,79 @@
+# The MIT License (MIT)
```

Comment:
Move and rename this this module to `xcube.server.testing` (`xcube/server/testing.py`)

Reply:
...and write a test for it :)

Reply:
done